src/api.js

```diff
@@ -251,10 +253,15 @@ Module["onRuntimeInitialized"] = function onRuntimeInitialized() {
             throw "Statement closed";
         }
         this["reset"]();
-        if (Array.isArray(values)) {
-            return this.bindFromArray(values);
+        var bindResult = true;
+        if (values) {
+            if (Array.isArray(values)) {
+                bindResult = this.bindFromArray(values);
+            } else {
+                bindResult = this.bindFromObject(values);
+            }
         }
-        return this.bindFromObject(values);
+        return bindResult;
```

Comment:
Maybe this would be clearer : 


```js
if (Array.isArray(values)) {
    return this.bindFromArray(values);
} else if (typeof values === "object") {
    return this.bindFromObject(values);
} else {
    return true;
}
```

Reply:
eslint seems to be not agree with you ... 
`error  Unnecessary 'else' after 'return'  no-else-return`

Reply:
Ok, 

```js
if (Array.isArray(values)) {
    return this.bindFromArray(values);
} else if (typeof values === "object") {
    return this.bindFromObject(values);
}
return true;
```

then

Reply:
minor [jslint] nit, but don't need the `else`

```
if (Array.isArray(values)) {
    return this.bindFromArray(values);
}
if (typeof values === "object") {
    return this.bindFromObject(values);
}
return true;
```

Reply:
No, the else is needed to avoid running the second check if the first one passed.

Reply:
eslint again  ... 
```javascript
        if (Array.isArray(values)) {
            return this.bindFromArray(values);
        } if (typeof values === "object") {
            return this.bindFromObject(values);
        }
        return true;
```
seems to be OK

Reply:
if the first one passed function end with return so else is not really needed (that's why eslint cry)...
Let me know what is fit to you (first option is still valid ....)

Reply:
OK, then 🙂

Reply:
hehe no worries ; b